packages/ui-library/src/theme/variants/chip-variants.ts

```diff
@@ -1,13 +1,18 @@
 import type { Components } from '@mui/material/styles';
 import { colors } from '../colors';
+import { SPACING } from '../constants';
 
 export const buildChipVariant = (mode: 'light' | 'dark'): Components => ({
   MuiChip: {
     styleOverrides: {
+      label: {
+        padding: 0,
+      },
       colorPrimary: {
         color: colors[mode].typography.typo2,
         backgroundColor: colors[mode].background.tertiary,
         border: `1.5px solid ${colors[mode].border.border2}`,
+        padding: `${SPACING(1)} ${SPACING(2)}`,
```

Comment:
its weird that by color we are adding padding, shouldnt this be general to the component?

Reply:
Yes, you are right. Applying it to `root` now